modules/virt-creating-a-service-from-a-virtual-machine.adoc

```diff
@@ -45,33 +44,31 @@ Labels on a virtual machine are passed through to the pod. The `special: key` la
 apiVersion: v1
 kind: Service
 metadata:
-  name: vmservice <1>
-  namespace: example-namespace <2>
+  name: example_service
+  namespace: example_namespace
 spec:
-  externalTrafficPolicy: Cluster <3>
+  externalTrafficPolicy: Cluster <1>
   ports:
-  - nodePort: 30000 <4>
+  - nodePort: 30000 <2>
```

Comment:
I would delete this to make sure it is selected by Kubernetes. Requesting a specific port by the end user is error prone.